index.html

```diff
@@ -1,111 +1,113 @@
-                                type="checkbox" />City Meetups</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Wiki"
-                                    type="checkbox" />Wiki</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Forum"
-                                        type="checkbox" />Forum</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Additional Courses"
-                                            type="checkbox" />Additional Courses</label>
-                    
-            </div>
+      <form id="survey-form" class="col-md-12  ">
+        <div class="form-group my-4">
+          <label class="mb-3" id="name-label" for="name">Name</label>
+          <input class="form-control" required type="text" id="name" placeholder="Enter your name" />
+        </div>
+        <div class="form-group my-4">
+          <label class="mb-3" id="email-label" for="email">Email</label>
+          <input class="form-control" required type="email" id="email" placeholder="Enter your email" />
+        </div>
+        <div class="form-group my-4">
+          <label class="mb-3" id="number-label" for="number">Age (optional)</label>
+          <input class="form-control" min="5" max="100" id="number" type="number" placeholder="Age" />
+        </div>
+        <div class="form-group my-4">
+          <label class="mb-3" for="dropdown">Which option best describes your current role?</label>
+          <select required class="form-control" id="dropdown">
+            <option disabled="" selected="" value="">
+              Select current role
+            </option>
+            <option value="student">Student</option>
+            <option value="job">Full Time Job</option>
+            <option value="learner">Full Time Learner</option>
+            <option value="preferNo">Prefer not to say</option>
+            <option value="other">Other</option>
+          </select>
+        </div>
+        <p class="mb-0">Would you recommend Microverse to a friend?</p>
+        <div class="form-check my-2">
+          <label class="form-check-label col-md-12 my-2" for="option-1"><input checked class="form-check-input"
+              id="option-1" value="Definitely" type="radio" name="recomandation" />Definitely</label>
+          <label class="form-check-label col-md-12 my-2" for="option-2"><input class="form-check-input" id="option-2"
+              value="Maybe" type="radio" name="recomandation" />
+            Maybe</label>
+          <label class="form-check-label col-md-12 my-2" for="option-3"><input class="form-check-input" id="option-3"
+              value="Not sure" type="radio" name="recomandation" />Not sure</label>
```

Comment:
Keep the indentation of two spaces so that it can make your code readable

Reply:
> 

- [ ] I would suggest while giving a code review. It is better to explain them kindly instead of giving direct orders.


> Example: I think it would be better if you could format your code properly to make them readable.